sdk/eventgrid/Azure.Messaging.EventGrid/src/autorest.md

```diff
@@ -50,5 +50,5 @@ input-file:
     -  https://github.com/Azure/azure-rest-api-specs/blob/00ac1cbffba123ba5e30cb324935100495d0700d/specification/eventgrid/data-plane/Microsoft.MachineLearningServices/stable/2018-01-01/MachineLearningServices.json
     -  https://github.com/Azure/azure-rest-api-specs/blob/00ac1cbffba123ba5e30cb324935100495d0700d/specification/eventgrid/data-plane/Microsoft.Cache/stable/2018-01-01/RedisCache.json
     -  https://github.com/Azure/azure-rest-api-specs/blob/00ac1cbffba123ba5e30cb324935100495d0700d/specification/eventgrid/data-plane/Microsoft.Web/stable/2018-01-01/Web.json
-    -  https://github.com/Azure/azure-rest-api-specs/blob/d587005c88aaa684cd744fe32019ad5f8f51dd98/specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json
+    -  https://github.com/Azure/azure-rest-api-specs/blob/3695f4ef3bf65c253509d2708c31c96b71822613/specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json
```

Comment:
Could we just use "master" now that all the changes have landed?

Reply:
I don't think so. My understanding is the .NET generator requires a permanent link to the file on GitHub.

Reply:
But you should switch to point to a single README.md file instead of individual swaggers.

Reply:
We can't because we are still using Matt's README for cloud events.

Reply:
It looks like my changes have been merged into master for azure-rest-api-specs, so this should no longer be a concern (https://github.com/Azure/azure-rest-api-specs/commit/5325e1398eb73e5de8c8a251909763b9e4803150)

Reply:
Ok will try and regenerate.